doc/design/cluster_train/data_dispatch.md

```diff
@@ -12,16 +12,18 @@
 在上图中显示了在一个实际生产环境中的应用（人脸识别）的数据流图。生产环境的日志数据会通过实时流的方式（Kafka）和离线数据的方式（HDFS）存储，并在集群中运行多个分布式数据处理任务，比如流式数据处理（online data process），离线批处理（offline data process）完成数据的预处理，提供给paddle作为训练数据。用于也可以上传labeled data到分布式存储补充训练数据。在paddle之上运行的深度学习训练输出的模型会提供给在线人脸识别的应用使用。
 
 ### 训练数据的存储
+We select CephFS to store our data.
 
-选择CephFS作为训练数据的存储服务。
+From the perspective of user program running in a Pod, it is mounted locally, as 
```

Comment:
Kubernetes will mount the CephFS path into the Pod which user's code running in.